app/components/popups/textdialog.py

```diff
@@ -0,0 +1,78 @@
+import cutlet
+import argostranslate.package as package
+import argostranslate.translate
+
+from PyQt5.QtWidgets import QLabel, QTextEdit, QVBoxLayout, QDialog
+from PyQt5 import QtCore
+
+class TextDialog(QDialog):
+    """
+    This "window" is a QWidget. If it has no parent, it
+    will appear as a free-floating window as we want.
+    """
+    def __init__(self):
+        super().__init__()
+        layout = QVBoxLayout()
+
+        self.label_ocr = QLabel("OCR")
```

Comment:
Please use camelCase names, since PyQt uses the same format. This includes the other variables in snake_case